src/screencast/screencast.c

```diff
@@ -109,6 +109,8 @@ bool setup_outputs(struct xdpw_screencast_context *ctx, struct xdpw_session *ses
 		return false;
 	}
 
+	// Disable screencast sharing to avoid sharing between dmabuf and shm capable clients
+	/*
```

Comment:
Hm. Arguably it's better to fallback to a single shm stream, than have two streams (one dmabuf and one shm). Not sure.

Reply:
Last time I checked, the first stream would set the format and the second one would either work or not with that, not triggering renegotiation. Can test this again.

Reply:
Ah that's a shame, if that's still the case consider opening a pipewire bug?

Reply:
Yeah, probably now is the time. There were some ideas to solve this. Renegotiating a existing stream has to be done in a way to not disturb the existing one, if they are incompatible. Another idea would be to negotiate a second stream, check if formats are compatible and tell PipeWire to reconnect the new stream to another old one, but I don't know the state.

Reply:
Tested this again and even triggering renegotiation after the second stream was connected didn't end in a setting working for both streams. 
Imho doing multiple screencasts at the same time is far less common than only one per output. Even if you are using multiple plugin instances of obs, the xdpw loop will pause, when the obs scene is hidden. 

Will open an issue on the PipeWire Project